integration_tests/README.md

```diff
@@ -0,0 +1,40 @@
+# dbt-data-reliability dbt Package Tests
```

Comment:
Cool!
I imagine this is for contributors? 
Should we update here as well? 
https://docs.elementary-data.com/general/contributions

Reply:
Generally yes, but at the moment I don't think so because we have a task this sprint to improve our testing infrastructure in which hopefully we'll have something more systematic than our current script.